src/mmw/apps/home/views.py

```diff
@@ -103,12 +104,17 @@ def get_model_packages():
 
 def create_layer_config_with_urls():
     layer_config_with_urls = deepcopy(settings.LAYER_GROUPS)
-    [[layer.update({'url': get_layer_url(layer)}) for layer in layers
+    layers = list(itertools.chain.from_iterable(
+        layer_config_with_urls.itervalues()))
```

Comment:
The values of the layer config are lists, so`.values()` returns a list of lists. I'm using the cast and iterator to flatten it.

I can change this to `create_layer_config_with_urls(layer_type)` and call it for each layer type (`base_maps`, `streams`, etc). That way we avoid having to deal with the list of lists at all.